src/lib.rs

```diff
@@ -271,20 +272,20 @@ impl HttpApiProblem {
     /// ```rust
     /// use http_api_problem::*;
     ///
-    /// let p = HttpApiProblem::with_title_from_status(404);
+    /// let p = HttpApiProblem::with_title_from_status(StatusCode::NOT_FOUND);
     ///
     /// assert_eq!(None, p.type_url);
-    /// assert_eq!(Some(HttpStatusCode::NotFound), p.status);
+    /// assert_eq!(Some(StatusCode::NOT_FOUND), p.status);
     /// assert_eq!("Not Found", p.title);
     /// assert_eq!(None, p.detail);
     /// assert_eq!(None, p.instance);
     /// ```
-    pub fn with_title_from_status<T: Into<HttpStatusCode>>(status: T) -> HttpApiProblem {
+    pub fn with_title_from_status<T: Into<StatusCode>>(status: T) -> HttpApiProblem {
         let status = status.into();
         HttpApiProblem {
             type_url: None,
             status: Some(status.into()),
-            title: status.title().to_string(),
+            title: status.canonical_reason().expect("unknown status code").to_string(),
```

Comment:
Shouldn't we use the `unwrap_or` as in line 261 to avoid a panic?

```suggestion
            title: status.canonical_reason().unwrap_or("<unknown status code>").to_string(),
```

Reply:
Yes definitely! That was a last-minute change before I pushed and I forgot this one :D